.gitlab-ci.yml

```diff
@@ -31,7 +31,7 @@ test:
     environment:
         name: $ENVIRONMENT_NAME
     script:
-        - mvn package --projects 'eva-accession-core,eva-accession-ws,eva-accession-clustering ' --settings .gitlab.settings.xml -P $MAVEN_PROFILE -DskipTests
+        - mvn package --projects 'eva-accession-core,eva-accession-ws,eva-accession-clustering ' --settings .gitlab.settings.xml -P $MAVEN_PROFILE -DskipTests -Dtimestamp="$DATETIME"
```

Comment:
do you think it's necessary to specify this in gitlab? I think maven will fill it on its own with the changes you did in the pom

Reply:
For easy debugging, I wanted the domain name to be consistent with the WAR file name generated from Gitlab.